apps/server/src/trpc/login.ts

```diff
@@ -0,0 +1,34 @@
+import { flatten } from '@modtree/utils'
+import { z } from 'zod'
+import { api } from '../main'
+import { entities } from '../schemas/entities'
+import { createRouter } from './router'
+
+export const login = createRouter()
+  /**
+   * user login
+   */
+  .mutation('login', {
```

Comment:
`/user/login` -> `/login`, to follow `entity/{entityId}/action` convention. I wouldn't mind having `{email}` for the middle part, but email doesn't work well with path params

Reply:
following the `entity/{entityId}/action` convention, it will be `/user/{userId}/login`